From: =?utf-8?q?Fabian_Gr=C3=BCnbichler?= <f.gruenbichler@proxmox.com>
Date: Wed, 6 Sep 2023 13:23:24 -0600
Subject: d-0021-vendor-remove-windows-dependencies
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

use something like

 find vendor -iname Cargo.toml -exec grep -H -n -e 'schannel' -e 'windows-sys' -e 'winapi' -e 'ntapi' -e 'wincon' -e 'winreg' -e 'windows' {} \;

to find dependencies on windows targets in vendored crates. you will likely
need to remove some hunks from this patch after pruning dependencies, since
hopefully a few of the crates patched during early rebasing are eliminated.

windows-bindgen and windows-metadata should not be removed, they are needed for
the build and don't pull in windows-sys and friends.

Forwarded: not-needed

Signed-off-by: Fabian Grünbichler <f.gruenbichler@proxmox.com>
---
 vendor/android-tzdata-0.1.1/Cargo.toml       |  1 +
 vendor/ansi_term-0.12.1/Cargo.toml           |  3 --
 vendor/anstream-0.6.13/Cargo.toml            |  7 -----
 vendor/anstyle-query-1.0.2/Cargo.toml        |  6 ----
 vendor/backtrace-0.3.71/Cargo.toml           | 14 ----------
 vendor/chrono-0.4.38/Cargo.toml              |  9 +-----
 vendor/colored-2.1.0/Cargo.toml              |  7 -----
 vendor/console-0.15.8/Cargo.toml             | 12 --------
 vendor/ctrlc-3.4.4/Cargo.toml                | 18 ------------
 vendor/curl-0.4.46/Cargo.toml                | 11 --------
 vendor/curl-sys-0.4.72+curl-8.6.0/Cargo.toml |  7 -----
 vendor/dbus-0.9.7/Cargo.toml                 |  4 ---
 vendor/dirs-sys-0.3.7/Cargo.toml             |  3 --
 vendor/dirs-sys-next-0.1.2/Cargo.toml        |  3 --
 vendor/dlmalloc-0.2.6/Cargo.toml             |  9 ------
 vendor/env_logger-0.11.3/Cargo.toml          |  2 +-
 vendor/errno-0.3.8/Cargo.toml                |  6 ----
 vendor/fd-lock-4.0.2/Cargo.toml              |  8 ------
 vendor/filetime-0.2.23/Cargo.toml            |  7 -----
 vendor/gix-sec-0.10.6/Cargo.toml             | 13 ---------
 vendor/home-0.5.9/Cargo.toml                 |  8 ------
 vendor/iana-time-zone-0.1.60/Cargo.toml      |  3 --
 vendor/ignore-0.4.20/Cargo.toml              |  3 --
 vendor/ignore-0.4.22/Cargo.toml              |  2 --
 vendor/is-terminal-0.4.12/Cargo.toml         | 11 --------
 vendor/libloading-0.8.3/Cargo.toml           |  7 -----
 vendor/libssh2-sys-0.3.0/Cargo.toml          |  8 ------
 vendor/mio-0.8.11/Cargo.toml                 | 11 --------
 vendor/native-tls-0.2.11/Cargo.toml          |  3 --
 vendor/nu-ansi-term-0.46.0/Cargo.toml        | 10 -------
 vendor/nu-ansi-term-0.49.0/Cargo.toml        | 10 -------
 vendor/opener-0.5.2/Cargo.toml               |  4 ---
 vendor/opener-0.6.1/Cargo.toml               |  9 ------
 vendor/opener-0.7.0/Cargo.toml               | 12 --------
 vendor/os_info-3.8.2/Cargo.toml              | 12 --------
 vendor/parking_lot_core-0.9.9/Cargo.toml     |  3 --
 vendor/reqwest-0.11.27/Cargo.toml            |  6 ----
 vendor/rustix-0.38.32/Cargo.toml             | 17 ------------
 vendor/same-file-1.0.6/Cargo.toml            |  2 --
 vendor/snapbox-0.5.9/Cargo.toml              |  6 ----
 vendor/socket2-0.5.6/Cargo.toml              | 10 -------
 vendor/stacker-0.1.15/Cargo.toml             | 10 -------
 vendor/sysinfo-0.30.8/Cargo.toml             | 41 ----------------------------
 vendor/tempfile-3.10.1/Cargo.toml            |  7 -----
 vendor/term-0.7.0/Cargo.toml                 |  6 ----
 vendor/termcolor-1.4.1/Cargo.toml            |  3 --
 vendor/terminal_size-0.3.0/Cargo.toml        |  7 -----
 vendor/termize-0.1.1/Cargo.toml              |  3 --
 vendor/tokio-1.37.0/Cargo.toml               | 21 --------------
 vendor/tokio-native-tls-0.3.1/Cargo.toml     | 17 ------------
 vendor/uuid-1.8.0/Cargo.toml                 |  1 -
 vendor/walkdir-2.5.0/Cargo.toml              |  3 --
 vendor/yansi-term-0.1.2/Cargo.toml           |  3 --
 53 files changed, 3 insertions(+), 426 deletions(-)

diff --git a/vendor/android-tzdata-0.1.1/Cargo.toml b/vendor/android-tzdata-0.1.1/Cargo.toml
index 805128a..0682717 100644
--- a/vendor/android-tzdata-0.1.1/Cargo.toml
+++ b/vendor/android-tzdata-0.1.1/Cargo.toml
@@ -32,3 +32,4 @@ repository = "https://github.com/RumovZ/android-tzdata"
 
 [dev-dependencies.zip]
 version = "0.6.4"
+repository = "https://github.com/rust-cli/concolor"
diff --git a/vendor/ansi_term-0.12.1/Cargo.toml b/vendor/ansi_term-0.12.1/Cargo.toml
index 0e5feba..3256c75 100644
--- a/vendor/ansi_term-0.12.1/Cargo.toml
+++ b/vendor/ansi_term-0.12.1/Cargo.toml
@@ -38,6 +38,3 @@ version = "1.0.39"
 
 [features]
 derive_serde_style = ["serde"]
-[target."cfg(target_os=\"windows\")".dependencies.winapi]
-version = "0.3.4"
-features = ["consoleapi", "errhandlingapi", "fileapi", "handleapi", "processenv"]
diff --git a/vendor/anstream-0.6.13/Cargo.toml b/vendor/anstream-0.6.13/Cargo.toml
index 94b329d..de559f1 100644
--- a/vendor/anstream-0.6.13/Cargo.toml
+++ b/vendor/anstream-0.6.13/Cargo.toml
@@ -130,11 +130,4 @@ version = "0.2.0"
 auto = ["dep:anstyle-query"]
 default = [
     "auto",
-    "wincon",
 ]
-test = []
-wincon = ["dep:anstyle-wincon"]
-
-[target."cfg(windows)".dependencies.anstyle-wincon]
-version = "3.0.1"
-optional = true
diff --git a/vendor/anstyle-query-1.0.2/Cargo.toml b/vendor/anstyle-query-1.0.2/Cargo.toml
index 63fead3..b3cc66e 100644
--- a/vendor/anstyle-query-1.0.2/Cargo.toml
+++ b/vendor/anstyle-query-1.0.2/Cargo.toml
@@ -72,9 +72,3 @@ replace = """
 [Unreleased]: https://github.com/rust-cli/anstyle/compare/{{tag_name}}...HEAD"""
 search = "<!-- next-url -->"
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52.0"
-features = [
-    "Win32_System_Console",
-    "Win32_Foundation",
-]
diff --git a/vendor/backtrace-0.3.71/Cargo.toml b/vendor/backtrace-0.3.71/Cargo.toml
index 6ac3a6d..0749f23 100644
--- a/vendor/backtrace-0.3.71/Cargo.toml
+++ b/vendor/backtrace-0.3.71/Cargo.toml
@@ -105,16 +105,6 @@ serialize-serde = ["serde"]
 std = []
 unix-backtrace = []
 verify-winapi = [
-    "winapi/dbghelp",
-    "winapi/handleapi",
-    "winapi/libloaderapi",
-    "winapi/memoryapi",
-    "winapi/minwindef",
-    "winapi/processthreadsapi",
-    "winapi/synchapi",
-    "winapi/tlhelp32",
-    "winapi/winbase",
-    "winapi/winnt",
 ]
 
 [target."cfg(not(all(windows, target_env = \"msvc\", not(target_vendor = \"uwp\"))))".dependencies.addr2line]
@@ -141,7 +131,3 @@ features = [
     "archive",
 ]
 default-features = false
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.9"
-optional = true
diff --git a/vendor/chrono-0.4.38/Cargo.toml b/vendor/chrono-0.4.38/Cargo.toml
index 1971471..e15e312 100644
--- a/vendor/chrono-0.4.38/Cargo.toml
+++ b/vendor/chrono-0.4.38/Cargo.toml
@@ -88,7 +88,7 @@ version = "1"
 __internal_bench = []
 alloc = []
 clock = [
-    "winapi",
+    "std",
     "iana-time-zone",
     "android-tzdata",
     "now",
@@ -125,7 +125,6 @@ wasmbind = [
     "wasm-bindgen",
     "js-sys",
 ]
-winapi = ["windows-targets"]
 
 [target."cfg(all(target_arch = \"wasm32\", not(any(target_os = \"emscripten\", target_os = \"wasi\"))))".dependencies.js-sys]
 version = "0.3"
@@ -147,9 +146,3 @@ version = "0.1.45"
 features = ["fallback"]
 optional = true
 
-[target."cfg(windows)".dependencies.windows-targets]
-version = "0.52"
-optional = true
-
-[target."cfg(windows)".dev-dependencies.windows-bindgen]
-version = "0.56"
diff --git a/vendor/colored-2.1.0/Cargo.toml b/vendor/colored-2.1.0/Cargo.toml
index c2134d7..9399b38 100644
--- a/vendor/colored-2.1.0/Cargo.toml
+++ b/vendor/colored-2.1.0/Cargo.toml
@@ -39,10 +39,3 @@ version = "=1.0.0-beta.3"
 
 [features]
 no-color = []
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.48"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Console",
-]
diff --git a/vendor/console-0.15.8/Cargo.toml b/vendor/console-0.15.8/Cargo.toml
index ecc237c..5e47717 100644
--- a/vendor/console-0.15.8/Cargo.toml
+++ b/vendor/console-0.15.8/Cargo.toml
@@ -58,15 +58,3 @@ default = [
     "ansi-parsing",
 ]
 windows-console-colors = ["ansi-parsing"]
-
-[target."cfg(windows)".dependencies.encode_unicode]
-version = "0.3"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52.0"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Console",
-    "Win32_Storage_FileSystem",
-    "Win32_UI_Input_KeyboardAndMouse",
-]
diff --git a/vendor/ctrlc-3.4.4/Cargo.toml b/vendor/ctrlc-3.4.4/Cargo.toml
index cd496d1..414adee 100644
--- a/vendor/ctrlc-3.4.4/Cargo.toml
+++ b/vendor/ctrlc-3.4.4/Cargo.toml
@@ -56,23 +56,5 @@ features = [
 ]
 default-features = false
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Threading",
-    "Win32_Security",
-    "Win32_System_Console",
-]
-
-[target."cfg(windows)".dev-dependencies.windows-sys]
-version = "0.52"
-features = [
-    "Win32_Storage_FileSystem",
-    "Win32_Foundation",
-    "Win32_System_IO",
-    "Win32_System_Console",
-]
-
 [badges.maintenance]
 status = "passively-maintained"
diff --git a/vendor/curl-0.4.46/Cargo.toml b/vendor/curl-0.4.46/Cargo.toml
index fc961dd..d102279 100644
--- a/vendor/curl-0.4.46/Cargo.toml
+++ b/vendor/curl-0.4.46/Cargo.toml
@@ -116,17 +116,6 @@ optional = true
 version = "0.9.64"
 optional = true
 
-[target."cfg(target_env = \"msvc\")".dependencies.schannel]
-version = "0.1.13"
-
-[target."cfg(target_env = \"msvc\")".dependencies.windows-sys]
-version = "0.52"
-features = [
-    "Win32_Foundation",
-    "Win32_System_LibraryLoader",
-    "Win32_Security_Cryptography",
-]
-
 [badges.appveyor]
 repository = "alexcrichton/curl-rust"
 
diff --git a/vendor/curl-sys-0.4.72+curl-8.6.0/Cargo.toml b/vendor/curl-sys-0.4.72+curl-8.6.0/Cargo.toml
index ea20c03..0751f3d 100644
--- a/vendor/curl-sys-0.4.72+curl-8.6.0/Cargo.toml
+++ b/vendor/curl-sys-0.4.72+curl-8.6.0/Cargo.toml
@@ -73,13 +73,6 @@ zlib-ng-compat = [
 version = "0.9.64"
 optional = true
 
-[target."cfg(target_env = \"msvc\")".build-dependencies.vcpkg]
-version = "0.2"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52"
-features = ["Win32_Networking_WinSock"]
-
 [badges.appveyor]
 repository = "alexcrichton/curl-rust"
 
diff --git a/vendor/dbus-0.9.7/Cargo.toml b/vendor/dbus-0.9.7/Cargo.toml
index d9f6d84..e04d90b 100644
--- a/vendor/dbus-0.9.7/Cargo.toml
+++ b/vendor/dbus-0.9.7/Cargo.toml
@@ -63,9 +63,5 @@ no-string-validation = []
 stdfd = []
 vendored = ["libdbus-sys/vendored"]
 
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.0"
-features = ["winsock2"]
-
 [badges.maintenance]
 status = "actively-developed"
diff --git a/vendor/dirs-sys-0.3.7/Cargo.toml b/vendor/dirs-sys-0.3.7/Cargo.toml
index 9951d24..0a86f4d 100644
--- a/vendor/dirs-sys-0.3.7/Cargo.toml
+++ b/vendor/dirs-sys-0.3.7/Cargo.toml
@@ -22,6 +22,3 @@ version = "0.4"
 default-features = false
 [target."cfg(unix)".dependencies.libc]
 version = "0.2"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["knownfolders", "objbase", "shlobj", "winbase", "winerror"]
diff --git a/vendor/dirs-sys-next-0.1.2/Cargo.toml b/vendor/dirs-sys-next-0.1.2/Cargo.toml
index e9d8d0c..acb2eb7 100644
--- a/vendor/dirs-sys-next-0.1.2/Cargo.toml
+++ b/vendor/dirs-sys-next-0.1.2/Cargo.toml
@@ -25,8 +25,5 @@ version = "0.4.0"
 default-features = false
 [target."cfg(unix)".dependencies.libc]
 version = "0.2"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["knownfolders", "objbase", "shlobj", "winbase", "winerror"]
 [badges.maintenance]
 status = "as-is"
diff --git a/vendor/dlmalloc-0.2.6/Cargo.toml b/vendor/dlmalloc-0.2.6/Cargo.toml
index dbc517b..dfc6b0e 100644
--- a/vendor/dlmalloc-0.2.6/Cargo.toml
+++ b/vendor/dlmalloc-0.2.6/Cargo.toml
@@ -62,12 +62,3 @@ rustc-dep-of-std = [
 [target."cfg(all(unix, not(target_arch = \"wasm32\")))".dependencies.libc]
 version = "0.2"
 default-features = false
-
-[target."cfg(target_os = \"windows\")".dependencies.windows-sys]
-version = "0.52.0"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Memory",
-    "Win32_System_Threading",
-    "Win32_System_SystemInformation",
-]
diff --git a/vendor/env_logger-0.11.3/Cargo.toml b/vendor/env_logger-0.11.3/Cargo.toml
index 8ef5b93..44ae34a 100644
--- a/vendor/env_logger-0.11.3/Cargo.toml
+++ b/vendor/env_logger-0.11.3/Cargo.toml
@@ -99,7 +99,7 @@ harness = false
 
 [dependencies.anstream]
 version = "0.6.11"
-features = ["wincon"]
+features = []
 optional = true
 default-features = false
 
diff --git a/vendor/errno-0.3.8/Cargo.toml b/vendor/errno-0.3.8/Cargo.toml
index e1d0bf5..60eeb86 100644
--- a/vendor/errno-0.3.8/Cargo.toml
+++ b/vendor/errno-0.3.8/Cargo.toml
@@ -41,9 +41,3 @@ default-features = false
 version = "0.2"
 default-features = false
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Diagnostics_Debug",
-]
diff --git a/vendor/fd-lock-4.0.2/Cargo.toml b/vendor/fd-lock-4.0.2/Cargo.toml
index 88826bc..fbf815d 100644
--- a/vendor/fd-lock-4.0.2/Cargo.toml
+++ b/vendor/fd-lock-4.0.2/Cargo.toml
@@ -43,11 +43,3 @@ version = "3.0.8"
 [target."cfg(unix)".dependencies.rustix]
 version = "0.38.0"
 features = ["fs"]
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52.0"
-features = [
-    "Win32_Foundation",
-    "Win32_Storage_FileSystem",
-    "Win32_System_IO",
-]
diff --git a/vendor/filetime-0.2.23/Cargo.toml b/vendor/filetime-0.2.23/Cargo.toml
index 0540ffb..2c65fa5 100644
--- a/vendor/filetime-0.2.23/Cargo.toml
+++ b/vendor/filetime-0.2.23/Cargo.toml
@@ -38,10 +38,3 @@ version = "0.4.1"
 
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.27"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52.0"
-features = [
-    "Win32_Foundation",
-    "Win32_Storage_FileSystem",
-]
diff --git a/vendor/gix-sec-0.10.6/Cargo.toml b/vendor/gix-sec-0.10.6/Cargo.toml
index df523e8..da37840 100644
--- a/vendor/gix-sec-0.10.6/Cargo.toml
+++ b/vendor/gix-sec-0.10.6/Cargo.toml
@@ -57,16 +57,3 @@ serde = [
 
 [target."cfg(not(windows))".dependencies.libc]
 version = "0.2.123"
-
-[target."cfg(windows)".dependencies.gix-path]
-version = "^0.10.7"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52.0"
-features = [
-    "Win32_Foundation",
-    "Win32_Security_Authorization",
-    "Win32_Storage_FileSystem",
-    "Win32_System_Memory",
-    "Win32_System_Threading",
-]
diff --git a/vendor/home-0.5.9/Cargo.toml b/vendor/home-0.5.9/Cargo.toml
index 93e3432..3bb226f 100644
--- a/vendor/home-0.5.9/Cargo.toml
+++ b/vendor/home-0.5.9/Cargo.toml
@@ -28,14 +28,6 @@ readme = "README.md"
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/rust-lang/cargo"
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52"
-features = [
-    "Win32_Foundation",
-    "Win32_UI_Shell",
-    "Win32_System_Com",
-]
-
 [lints.clippy]
 dbg_macro = "warn"
 disallowed_methods = "warn"
diff --git a/vendor/iana-time-zone-0.1.60/Cargo.toml b/vendor/iana-time-zone-0.1.60/Cargo.toml
index ad54889..5db5668 100644
--- a/vendor/iana-time-zone-0.1.60/Cargo.toml
+++ b/vendor/iana-time-zone-0.1.60/Cargo.toml
@@ -52,6 +52,3 @@ version = "0.1.5"
 
 [target."cfg(target_os = \"haiku\")".dependencies.iana-time-zone-haiku]
 version = "0.1.1"
-
-[target."cfg(target_os = \"windows\")".dependencies.windows-core]
-version = ">=0.50, <=0.52"
diff --git a/vendor/ignore-0.4.20/Cargo.toml b/vendor/ignore-0.4.20/Cargo.toml
index b397dcf..2022a87 100644
--- a/vendor/ignore-0.4.20/Cargo.toml
+++ b/vendor/ignore-0.4.20/Cargo.toml
@@ -64,6 +64,3 @@ version = "0.5.0"
 
 [features]
 simd-accel = ["globset/simd-accel"]
-
-[target."cfg(windows)".dependencies.winapi-util]
-version = "0.1.2"
diff --git a/vendor/ignore-0.4.22/Cargo.toml b/vendor/ignore-0.4.22/Cargo.toml
index d038e57..8a6dc8e 100644
--- a/vendor/ignore-0.4.22/Cargo.toml
+++ b/vendor/ignore-0.4.22/Cargo.toml
@@ -77,5 +77,3 @@ version = "0.5.8"
 [features]
 simd-accel = []
 
-[target."cfg(windows)".dependencies.winapi-util]
-version = "0.1.2"
diff --git a/vendor/is-terminal-0.4.12/Cargo.toml b/vendor/is-terminal-0.4.12/Cargo.toml
index 98a8829..01ce09a 100644
--- a/vendor/is-terminal-0.4.12/Cargo.toml
+++ b/vendor/is-terminal-0.4.12/Cargo.toml
@@ -57,14 +57,3 @@ features = ["stdio"]
 
 [target."cfg(target_os = \"hermit\")".dependencies.hermit-abi]
 version = "0.3.0"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52.0"
-features = [
-    "Win32_Foundation",
-    "Win32_Storage_FileSystem",
-    "Win32_System_Console",
-]
-
-[target."cfg(windows)".dev-dependencies.tempfile]
-version = "3"
diff --git a/vendor/libloading-0.8.3/Cargo.toml b/vendor/libloading-0.8.3/Cargo.toml
index d5e5548..2204533 100644
--- a/vendor/libloading-0.8.3/Cargo.toml
+++ b/vendor/libloading-0.8.3/Cargo.toml
@@ -42,10 +42,3 @@ version = "1.1"
 
 [target."cfg(unix)".dependencies.cfg-if]
 version = "1"
-
-[target."cfg(windows)".dependencies.windows-targets]
-version = ">=0.48, <0.53"
-
-[target."cfg(windows)".dev-dependencies.windows-sys]
-version = "0.52"
-features = ["Win32_Foundation"]
diff --git a/vendor/libssh2-sys-0.3.0/Cargo.toml b/vendor/libssh2-sys-0.3.0/Cargo.toml
index 45f4a71..516644c 100644
--- a/vendor/libssh2-sys-0.3.0/Cargo.toml
+++ b/vendor/libssh2-sys-0.3.0/Cargo.toml
@@ -43,16 +43,8 @@ version = "1.0.25"
 version = "0.3.11"
 
 [features]
-openssl-on-win32 = ["openssl-sys"]
 vendored-openssl = ["openssl-sys/vendored"]
 zlib-ng-compat = ["libz-sys/zlib-ng"]
 
-[target."cfg(target_env = \"msvc\")".build-dependencies.vcpkg]
-version = "0.2"
-
 [target."cfg(unix)".dependencies.openssl-sys]
 version = "0.9.35"
-
-[target."cfg(windows)".dependencies.openssl-sys]
-version = "0.9.35"
-optional = true
diff --git a/vendor/mio-0.8.11/Cargo.toml b/vendor/mio-0.8.11/Cargo.toml
index 5d112d9..467ab24 100644
--- a/vendor/mio-0.8.11/Cargo.toml
+++ b/vendor/mio-0.8.11/Cargo.toml
@@ -103,8 +103,6 @@ default = ["log"]
 net = []
 os-ext = [
     "os-poll",
-    "windows-sys/Win32_System_Pipes",
-    "windows-sys/Win32_Security",
 ]
 os-poll = []
 
@@ -117,12 +115,3 @@ version = "0.11.0"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.149"
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.48"
-features = [
-    "Win32_Foundation",
-    "Win32_Networking_WinSock",
-    "Win32_Storage_FileSystem",
-    "Win32_System_IO",
-    "Win32_System_WindowsProgramming",
-]
diff --git a/vendor/native-tls-0.2.11/Cargo.toml b/vendor/native-tls-0.2.11/Cargo.toml
index a059236..d36b44b 100644
--- a/vendor/native-tls-0.2.11/Cargo.toml
+++ b/vendor/native-tls-0.2.11/Cargo.toml
@@ -61,6 +61,3 @@ version = "0.1"
 
 [target."cfg(not(any(target_os = \"windows\", target_os = \"macos\", target_os = \"ios\")))".dependencies.openssl-sys]
 version = "0.9.55"
-
-[target."cfg(target_os = \"windows\")".dependencies.schannel]
-version = "0.1.17"
diff --git a/vendor/nu-ansi-term-0.46.0/Cargo.toml b/vendor/nu-ansi-term-0.46.0/Cargo.toml
index 209e055..aa40f02 100644
--- a/vendor/nu-ansi-term-0.46.0/Cargo.toml
+++ b/vendor/nu-ansi-term-0.46.0/Cargo.toml
@@ -45,13 +45,3 @@ version = "1.0.39"
 
 [features]
 derive_serde_style = ["serde"]
-
-[target."cfg(target_os=\"windows\")".dependencies.winapi]
-version = "0.3.4"
-features = [
-    "consoleapi",
-    "errhandlingapi",
-    "fileapi",
-    "handleapi",
-    "processenv",
-]
diff --git a/vendor/nu-ansi-term-0.49.0/Cargo.toml b/vendor/nu-ansi-term-0.49.0/Cargo.toml
index f36dade..3e86523 100644
--- a/vendor/nu-ansi-term-0.49.0/Cargo.toml
+++ b/vendor/nu-ansi-term-0.49.0/Cargo.toml
@@ -42,13 +42,3 @@ version = "1.0.94"
 [features]
 derive_serde_style = ["serde"]
 gnu_legacy = []
-
-[target."cfg(windows)".dependencies.windows]
-version = "0.48.0"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Console",
-    "Win32_Storage_FileSystem",
-    "Win32_Security",
-]
-package = "windows-sys"
diff --git a/vendor/opener-0.5.2/Cargo.toml b/vendor/opener-0.5.2/Cargo.toml
index 8d91b5e..2d7313b 100644
--- a/vendor/opener-0.5.2/Cargo.toml
+++ b/vendor/opener-0.5.2/Cargo.toml
@@ -32,10 +32,6 @@ version = "0.9"
 [target."cfg(target_os = \"linux\")".dependencies.bstr]
 version = "1"
 
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["shellapi"]
-
 [badges.appveyor]
 branch = "master"
 repository = "Seeker14491/opener"
diff --git a/vendor/opener-0.6.1/Cargo.toml b/vendor/opener-0.6.1/Cargo.toml
index 79b4be9..69cc269 100644
--- a/vendor/opener-0.6.1/Cargo.toml
+++ b/vendor/opener-0.6.1/Cargo.toml
@@ -40,8 +40,6 @@ version = "0.9"
 reveal = [
     "dep:url",
     "dep:dbus",
-    "winapi/shtypes",
-    "winapi/objbase",
 ]
 
 [target."cfg(target_os = \"linux\")".dependencies.bstr]
@@ -56,13 +54,6 @@ optional = true
 version = "2"
 optional = true
 
-[target."cfg(windows)".dependencies.normpath]
-version = "1"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["shellapi"]
-
 [badges.appveyor]
 branch = "master"
 repository = "Seeker14491/opener"
diff --git a/vendor/opener-0.7.0/Cargo.toml b/vendor/opener-0.7.0/Cargo.toml
index 46739db..df53740 100644
--- a/vendor/opener-0.7.0/Cargo.toml
+++ b/vendor/opener-0.7.0/Cargo.toml
@@ -39,7 +39,6 @@ default = ["dbus-vendored"]
 reveal = [
     "dep:url",
     "dep:dbus",
-    "windows-sys/Win32_System_Com",
 ]
 
 [target."cfg(target_os = \"linux\")".dependencies.bstr]
@@ -53,16 +52,5 @@ optional = true
 version = "2"
 optional = true
 
-[target."cfg(windows)".dependencies.normpath]
-version = "1"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52"
-features = [
-    "Win32_Foundation",
-    "Win32_UI_Shell",
-    "Win32_UI_WindowsAndMessaging",
-]
-
 [badges.maintenance]
 status = "passively-maintained"
diff --git a/vendor/os_info-3.8.2/Cargo.toml b/vendor/os_info-3.8.2/Cargo.toml
index 9aa2067..c1a79b8 100644
--- a/vendor/os_info-3.8.2/Cargo.toml
+++ b/vendor/os_info-3.8.2/Cargo.toml
@@ -48,15 +48,3 @@ version = "1"
 
 [features]
 default = ["serde"]
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52"
-features = [
-    "Win32_Foundation",
-    "Win32_System_LibraryLoader",
-    "Win32_System_Registry",
-    "Win32_System_SystemInformation",
-    "Win32_System_SystemServices",
-    "Win32_System_Threading",
-    "Win32_UI_WindowsAndMessaging",
-]
diff --git a/vendor/parking_lot_core-0.9.9/Cargo.toml b/vendor/parking_lot_core-0.9.9/Cargo.toml
index 83d9f23..dbe1534 100644
--- a/vendor/parking_lot_core-0.9.9/Cargo.toml
+++ b/vendor/parking_lot_core-0.9.9/Cargo.toml
@@ -61,6 +61,3 @@ version = "0.4"
 
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.95"
-
-[target."cfg(windows)".dependencies.windows-targets]
-version = "0.48.0"
diff --git a/vendor/reqwest-0.11.27/Cargo.toml b/vendor/reqwest-0.11.27/Cargo.toml
index 6a6a648..32e2c9e 100644
--- a/vendor/reqwest-0.11.27/Cargo.toml
+++ b/vendor/reqwest-0.11.27/Cargo.toml
@@ -455,9 +455,3 @@ features = ["serde-serialize"]
 
 [target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]
 version = "0.3"
-
-[target."cfg(target_os = \"macos\")".dependencies.system-configuration]
-version = "0.5.1"
-
-[target."cfg(windows)".dependencies.winreg]
-version = "0.50.0"
diff --git a/vendor/rustix-0.38.32/Cargo.toml b/vendor/rustix-0.38.32/Cargo.toml
index 589611b..55de475 100644
--- a/vendor/rustix-0.38.32/Cargo.toml
+++ b/vendor/rustix-0.38.32/Cargo.toml
@@ -253,20 +253,3 @@ package = "errno"
 [target."cfg(any(target_os = \"android\", target_os = \"linux\"))".dependencies.once_cell]
 version = "1.5.2"
 optional = true
-
-[target."cfg(windows)".dependencies.libc_errno]
-version = "0.3.8"
-default-features = false
-package = "errno"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52.0"
-features = [
-    "Win32_Foundation",
-    "Win32_Networking_WinSock",
-    "Win32_NetworkManagement_IpHelper",
-    "Win32_System_Threading",
-]
-
-[target."cfg(windows)".dev-dependencies.ctor]
-version = "0.2.0"
diff --git a/vendor/same-file-1.0.6/Cargo.toml b/vendor/same-file-1.0.6/Cargo.toml
index 4f66820..11ef472 100644
--- a/vendor/same-file-1.0.6/Cargo.toml
+++ b/vendor/same-file-1.0.6/Cargo.toml
@@ -25,5 +25,3 @@ license = "Unlicense/MIT"
 repository = "https://github.com/BurntSushi/same-file"
 [dev-dependencies.doc-comment]
 version = "0.3"
-[target."cfg(windows)".dependencies.winapi-util]
-version = "0.1.1"
diff --git a/vendor/snapbox-0.5.9/Cargo.toml b/vendor/snapbox-0.5.9/Cargo.toml
index 9c80c0f..79e3485 100644
--- a/vendor/snapbox-0.5.9/Cargo.toml
+++ b/vendor/snapbox-0.5.9/Cargo.toml
@@ -169,7 +169,6 @@ cmd = [
     "dep:os_pipe",
     "dep:wait-timeout",
     "dep:libc",
-    "dep:windows-sys",
 ]
 color = [
     "dep:anstream",
@@ -211,8 +210,3 @@ term-svg = [
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.137"
 optional = true
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52.0"
-features = ["Win32_Foundation"]
-optional = true
diff --git a/vendor/socket2-0.5.6/Cargo.toml b/vendor/socket2-0.5.6/Cargo.toml
index 8d36ede..2fb6394 100644
--- a/vendor/socket2-0.5.6/Cargo.toml
+++ b/vendor/socket2-0.5.6/Cargo.toml
@@ -75,13 +75,3 @@ all = []
 
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.150"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52"
-features = [
-    "Win32_Foundation",
-    "Win32_Networking_WinSock",
-    "Win32_System_IO",
-    "Win32_System_Threading",
-    "Win32_System_WindowsProgramming",
-]
diff --git a/vendor/stacker-0.1.15/Cargo.toml b/vendor/stacker-0.1.15/Cargo.toml
index 160cbc0..ba02aac 100644
--- a/vendor/stacker-0.1.15/Cargo.toml
+++ b/vendor/stacker-0.1.15/Cargo.toml
@@ -43,13 +43,3 @@ version = "0.1.7"
 
 [build-dependencies.cc]
 version = "1.0.2"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.6"
-features = [
-    "memoryapi",
-    "winbase",
-    "fibersapi",
-    "processthreadsapi",
-    "minwindef",
-]
diff --git a/vendor/sysinfo-0.30.8/Cargo.toml b/vendor/sysinfo-0.30.8/Cargo.toml
index fc1b398..92387fc 100644
--- a/vendor/sysinfo-0.30.8/Cargo.toml
+++ b/vendor/sysinfo-0.30.8/Cargo.toml
@@ -78,44 +78,3 @@ version = "1.18"
 
 [target."cfg(not(any(target_os = \"unknown\", target_arch = \"wasm32\")))".dependencies.libc]
 version = "^0.2.153"
-
-[target."cfg(windows)".dependencies.ntapi]
-version = "0.4"
-
-[target."cfg(windows)".dependencies.windows]
-version = "0.52"
-features = [
-    "Wdk_System_SystemInformation",
-    "Wdk_System_SystemServices",
-    "Wdk_System_Threading",
-    "Win32_Foundation",
-    "Win32_NetworkManagement_IpHelper",
-    "Win32_NetworkManagement_NetManagement",
-    "Win32_NetworkManagement_Ndis",
-    "Win32_Networking_WinSock",
-    "Win32_Security",
-    "Win32_Security_Authentication_Identity",
-    "Win32_Security_Authorization",
-    "Win32_Storage_FileSystem",
-    "Win32_System_Com",
-    "Win32_System_Diagnostics_Debug",
-    "Win32_System_IO",
-    "Win32_System_Ioctl",
-    "Win32_System_LibraryLoader",
-    "Win32_System_Kernel",
-    "Win32_System_Memory",
-    "Win32_System_Ole",
-    "Win32_System_Performance",
-    "Win32_System_Power",
-    "Win32_System_ProcessStatus",
-    "Win32_System_Registry",
-    "Win32_System_RemoteDesktop",
-    "Win32_System_Rpc",
-    "Win32_System_SystemInformation",
-    "Win32_System_SystemServices",
-    "Win32_System_Threading",
-    "Win32_System_Variant",
-    "Win32_System_WindowsProgramming",
-    "Win32_System_Wmi",
-    "Win32_UI_Shell",
-]
diff --git a/vendor/tempfile-3.10.1/Cargo.toml b/vendor/tempfile-3.10.1/Cargo.toml
index ee1016f..88f6a24 100644
--- a/vendor/tempfile-3.10.1/Cargo.toml
+++ b/vendor/tempfile-3.10.1/Cargo.toml
@@ -47,10 +47,3 @@ nightly = []
 [target."cfg(any(unix, target_os = \"wasi\"))".dependencies.rustix]
 version = "0.38.31"
 features = ["fs"]
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52"
-features = [
-    "Win32_Storage_FileSystem",
-    "Win32_Foundation",
-]
diff --git a/vendor/term-0.7.0/Cargo.toml b/vendor/term-0.7.0/Cargo.toml
index e89261e..ad5d62b 100644
--- a/vendor/term-0.7.0/Cargo.toml
+++ b/vendor/term-0.7.0/Cargo.toml
@@ -28,12 +28,6 @@ version = "2"
 
 [features]
 default = []
-[target."cfg(windows)".dependencies.rustversion]
-version = "1"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["consoleapi", "wincon", "handleapi", "fileapi"]
 [badges.appveyor]
 repository = "Stebalien/term"
 
diff --git a/vendor/termcolor-1.4.1/Cargo.toml b/vendor/termcolor-1.4.1/Cargo.toml
index f90eaca..d32db67 100644
--- a/vendor/termcolor-1.4.1/Cargo.toml
+++ b/vendor/termcolor-1.4.1/Cargo.toml
@@ -35,6 +35,3 @@ name = "termcolor"
 bench = false
 
 [dev-dependencies]
-
-[target."cfg(windows)".dependencies.winapi-util]
-version = "0.1.3"
diff --git a/vendor/terminal_size-0.3.0/Cargo.toml b/vendor/terminal_size-0.3.0/Cargo.toml
index f810025..0fbe3a1 100644
--- a/vendor/terminal_size-0.3.0/Cargo.toml
+++ b/vendor/terminal_size-0.3.0/Cargo.toml
@@ -30,10 +30,3 @@ repository = "https://github.com/eminence/terminal-size"
 [target."cfg(not(windows))".dependencies.rustix]
 version = "0.38.0"
 features = ["termios"]
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.48.0"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Console",
-]
diff --git a/vendor/termize-0.1.1/Cargo.toml b/vendor/termize-0.1.1/Cargo.toml
index d248f4d..9bd9f37 100644
--- a/vendor/termize-0.1.1/Cargo.toml
+++ b/vendor/termize-0.1.1/Cargo.toml
@@ -49,8 +49,5 @@ rpath = false
 [dependencies]
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.66"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.8"
-features = ["handleapi", "processenv", "wincon", "winbase"]
 [badges.cirrus-ci]
 repository = "JohnTitor/termize"
diff --git a/vendor/tokio-1.37.0/Cargo.toml b/vendor/tokio-1.37.0/Cargo.toml
index 0826335..6273a66 100644
--- a/vendor/tokio-1.37.0/Cargo.toml
+++ b/vendor/tokio-1.37.0/Cargo.toml
@@ -130,11 +130,6 @@ net = [
     "mio/os-ext",
     "mio/net",
     "socket2",
-    "windows-sys/Win32_Foundation",
-    "windows-sys/Win32_Security",
-    "windows-sys/Win32_Storage_FileSystem",
-    "windows-sys/Win32_System_Pipes",
-    "windows-sys/Win32_System_SystemServices",
 ]
 process = [
     "bytes",
@@ -143,9 +138,6 @@ process = [
     "mio/os-ext",
     "mio/net",
     "signal-hook-registry",
-    "windows-sys/Win32_Foundation",
-    "windows-sys/Win32_System_Threading",
-    "windows-sys/Win32_System_WindowsProgramming",
 ]
 rt = []
 rt-multi-thread = [
@@ -158,8 +150,6 @@ signal = [
     "mio/net",
     "mio/os-ext",
     "signal-hook-registry",
-    "windows-sys/Win32_Foundation",
-    "windows-sys/Win32_System_Console",
 ]
 sync = []
 test-util = [
@@ -224,14 +214,3 @@ features = [
     "socket",
 ]
 default-features = false
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.48"
-optional = true
-
-[target."cfg(windows)".dev-dependencies.windows-sys]
-version = "0.48"
-features = [
-    "Win32_Foundation",
-    "Win32_Security_Authorization",
-]
diff --git a/vendor/tokio-native-tls-0.3.1/Cargo.toml b/vendor/tokio-native-tls-0.3.1/Cargo.toml
index 76be3d3..3ada811 100644
--- a/vendor/tokio-native-tls-0.3.1/Cargo.toml
+++ b/vendor/tokio-native-tls-0.3.1/Cargo.toml
@@ -76,20 +76,3 @@ version = "0.10"
 
 [target."cfg(any(target_os = \"macos\", target_os = \"ios\"))".dev-dependencies.security-framework]
 version = "0.2"
-
-[target."cfg(windows)".dev-dependencies.schannel]
-version = "0.1"
-
-[target."cfg(windows)".dev-dependencies.winapi]
-version = "0.3"
-features = [
-    "lmcons",
-    "basetsd",
-    "minwinbase",
-    "minwindef",
-    "ntdef",
-    "sysinfoapi",
-    "timezoneapi",
-    "wincrypt",
-    "winerror",
-]
diff --git a/vendor/uuid-1.8.0/Cargo.toml b/vendor/uuid-1.8.0/Cargo.toml
index 6bc7d113..e6df87a 100644
--- a/vendor/uuid-1.8.0/Cargo.toml
+++ b/vendor/uuid-1.8.0/Cargo.toml
@@ -197,7 +197,6 @@ version = "0.3"
 
 [target."cfg(target = \"wasm32-unknown-unknown\")".dev-dependencies.wasm-bindgen]
 version = "0.2"
-
 [badges.is-it-maintained-issue-resolution]
 repository = "uuid-rs/uuid"
 
diff --git a/vendor/walkdir-2.5.0/Cargo.toml b/vendor/walkdir-2.5.0/Cargo.toml
index 4fda2f4..1741677 100644
--- a/vendor/walkdir-2.5.0/Cargo.toml
+++ b/vendor/walkdir-2.5.0/Cargo.toml
@@ -39,9 +39,6 @@ version = "1.0.1"
 [dev-dependencies.doc-comment]
 version = "0.3"
 
-[target."cfg(windows)".dependencies.winapi-util]
-version = "0.1.1"
-
 [badges.appveyor]
 repository = "BurntSushi/walkdir"
 
diff --git a/vendor/yansi-term-0.1.2/Cargo.toml b/vendor/yansi-term-0.1.2/Cargo.toml
index 0317866..88ce8ef 100644
--- a/vendor/yansi-term-0.1.2/Cargo.toml
+++ b/vendor/yansi-term-0.1.2/Cargo.toml
@@ -36,9 +36,6 @@ version = "1.0"
 
 [features]
 derive_serde_style = ["serde"]
-[target."cfg(target_os=\"windows\")".dependencies.winapi]
-version = "0.3.4"
-features = ["consoleapi", "errhandlingapi", "fileapi", "handleapi", "processenv"]
 [badges.maintenance]
 status = "actively-developed"
 
